### PR TITLE
test build with updated github for apple silicon

### PR DESCRIPTION
**Pull request by benonymus**
I want to test out the updated [github](https://github.com/pulsar-edit/github/pull/2) library on a real build, to verify the fix.

### CODE REVIEW

**Comment by confused-Techie** [comment]
This looks fine to me, but as @mauricioszabo was the one to originally setup Cirrus, and is currently the one maintaining it, I'd like to defer to their opinion on this change, if it seems fine by them. Otherwise if there's no issue there, then lets get it merged

File: script/electron-builder.js [hunk 1]
```diff
@@ -68,81 +81,85 @@ let options = {
...
-    ]
+  win: {
+    icon: icoIcon,
+    target: [{ target: "nsis" }, { target: "portable" }],
   },
-  "extraMetadata": {
-  }
-}
+  extraMetadata: {},
+  asarUnpack: ["node_modules/github/bin/*"],
```
Comment (Sertonix) [inline script/electron-builder.js:121]: The `extraMetadata` is overwritten in the `main` function so this has no effect

Comment (benonymus) [reply script/electron-builder.js:121]: extraMetadata was here before, I only added asarUnpack

Comment (Sertonix) [reply script/electron-builder.js:121]: 🤦 I thought `asarUnpack` was inside `extraMetadata`. Sorry for that.

Comment (mauricioszabo) [reply script/electron-builder.js:121]: With the GitHub fixes, do we actually need this asarUnpack key?

**Comment by benonymus** [comment]
Yes we do. The GitHub fixes are for apple silicon support.

Bence Janos Toth
________________________________

**Comment by jonian** [comment]
@benonymus, you should open e new PR for `asarUnpack` since it resolves another issue not related to apple silicon.

**Comment by benonymus** [comment]
> @benonymus, you should open a new PR for `asarUnpack` since it resolves another issue not related to apple silicon.

Yeah that is fair. If we say this this is fine by others as well. 

This additional issue was only discovered after I made the fix for Apple Silicon.

It fixes this: https://discord.com/channels/992103415163396136/992103415163396139/1040564640062648370

**Comment by benonymus** [comment]
Technically this pr can be closed. I got what I wanted to try. 
I think we should merge the pr on the GitHub repo and then have a new tag with the fix there in Pulsar instead of my fork.

https://github.com/pulsar-edit/github/pull/2 

What do you think @mauricioszabo ?

**Comment by confused-Techie** [comment]
Like stated above that this should be closed, now especially as we have #186 merged, should we close this one @benonymus?

**Pull request closed without merging**